internal/dataplane/kong_client.go

```diff
@@ -461,14 +461,25 @@ func (c *KongClient) sendOutToGatewayClients(
 	ctx context.Context, s *kongstate.KongState, config sendconfig.Config,
 ) ([]string, error) {
 	gatewayClients := c.clientsProvider.GatewayClients()
-	c.logger.V(util.DebugLevel).Info("sending configuration to gateway clients", "count", len(gatewayClients))
+	previousSHAs := c.SHAs
+	if len(gatewayClients) == 0 {
+		c.logger.Error(
+			errors.New("no ready gateway clients"),
+			"could not send configuration to gateways because no ready gateway clients",
```

Comment:
```suggestion
			"Could not send the configuration to gateways",
```
we capitalize logs see https://github.com/Kong/kubernetes-ingress-controller/pull/4992, furthermore repeating the same stuff in message that is in the error seems redundant for me

Reply:
done.